packages/bruno-app/src/providers/ReduxStore/slices/collections/actions.js

```
@@ -549,6 +550,37 @@ export const wsConnectOnly = (item, collectionUid) => (dispatch, getState) => {
+      activeGlobalEnvironmentUid
+    });
+    collectionCopy.globalEnvironmentVariables = globalEnvironmentVariables;
+
+    const environment = findEnvironmentInCollection(collectionCopy, collectionCopy.activeEnvironmentUid);
+
+    connectSignalR(itemCopy, collectionCopy, environment, collectionCopy.runtimeVariables)
+      .then(resolve)
+      .catch((err) => {
+        toast.error(err.message);
+      });
+  });
```

Biome (2.5.1)

[error] Promise executor functions should not be `async`.

(lint/suspicious/noAsyncPromiseExecutor)